frontend/packages/knative-plugin/src/components/overview/serving-list/ServingListsPage.tsx

```diff
@@ -67,6 +70,7 @@ const ServingListPage: React.FC<ServingListPageProps> = ({ match }) => {
         match={match}
         title={t('knative-plugin~Serving')}
         menuActions={menuActions}
+        navMenu="Serving"
```

Comment:
Not essential to get this in, but I saw that the Serverless > Serving and Serverless Evening pages in the admin perspective use this for telemetry but not in the UI. I think that's fine, as it is like before. :+1: